### ISSUE TITLE
Avoid unnecessary PR

### ISSUE THREAD

**Issue by chrvadala** [dependabot/feedback#14]
Hi guys,
I think that when there's a new bump release on a package that has an already open PR, you can just upgrade that PR avoiding to open a new one.

Let's see this history, that happened in just 3 days:
Bump webpack from 4.1.1 to 4.6.0 https://github.com/chrvadala/react-svg-pan-zoom/pull/99
Superseded by https://github.com/chrvadala/react-svg-pan-zoom/pull/101

Bump webpack from 4.1.1 to 4.7.0 https://github.com/chrvadala/react-svg-pan-zoom/pull/101
Superseded by https://github.com/chrvadala/react-svg-pan-zoom/pull/106

Bump webpack from 4.1.1 to 4.8.1 https://github.com/chrvadala/react-svg-pan-zoom/pull/106

**Comment by greysteil** [comment]
Makes sense to me - I don't think anyone's particularly attached to the old PRs. It's not a totally trivial change, but I should be able to get to it pretty quickly.

Thanks for the feedback!

**Comment by AndreaGiardini** [comment]
It makes perfect sense to me too, one of the advantages is also that the assignee/reviewer are kept

**Comment by php-coder** [comment]
> It's not a totally trivial change, but I should be able to get to it pretty quickly.

@greysteil Is it already fixed?

**Comment by greysteil** [comment]
It's not - this is probably a full day's work on my side. I'll get to it, but have had a bunch of other higher priorities.

In the meantime, assignees and reviewers are now copied over from the old PRs. 😎

**Comment by lewisblackwood** [comment]
Another reason this would be nice to have is so that PR comments aren't lost when a new PR is opened. Sometimes I find myself leaving a PR comment to the effect of:

> This is a major version change and we should test out the breaking changes manually! 😬

When the PR gets superseded by a minor version bump, the comment gets left behind. It would be handy if you could mark comments that should be carried over, but I guess maintaining a single PR is even better!

**Comment by waghanza** [comment]
This could also avoid unnecessary `CI` **builds**

I mean generaly `CI` run on each `PR`, and having 2 `PR`s make the work twice on `CI` side

**Comment by JacobEvelyn** [comment]
@greysteil are you still working on this? I have the exact same problem as @lewisblackwood—if I have extra context on a gem I'll leave a comment in the PR, but if the PR is superseded that comment gets lost forever.

**Comment by merwok** [comment]
Thanks @feelepxyz for keeping all these tickets open despite zealous stalebot! :slightly_smiling_face:
